fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -647,10 +736,10 @@ private fun BottomDrawer(
             open = !drawerState.isClosed,
             onClose = onDismiss,
             fraction = {
-                if (drawerState.anchors.isEmpty()
-                ) {
+                if (drawerState.anchors.isEmpty() || drawerState.skipOpenState) {
```

Comment:
scrim need to be shown when the anchor is calculated. Remove the  drawerState.skipOpenState

Reply:
If you look at the else part, it searches for the anchor value corresponding to the OPEN state, and first time after skipOpenState = true has been set when this is being executed, anchors haven't been created yet, that throws NULL POINTER error.

Reply:
Okay, will handle that case in else part